MetaMask/metamask-extension
# Improve conflux logo in dark theme

## **Description**



Currently, Conflux eSpace logo is supported in MetaMask, but its display will become invisible in dark mode. This pull request would replace the original logo so it will be in well display in both themes.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/26219?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/23419

## **Manual testing steps**

1. Switch browser/system to dark theme
2. Visit ConfluxScan (https://evm.confluxscan.io/)
3. click "connect wallet" on the top toolbar
4. Will see "switch network" botton and click it 
5. Will see network switch prompt in Metamask. And can be seen if the conflux logo is visible or not

## **Screenshots/Recordings**



### **Before**

![image](https://github.com/user-attachments/assets/e7e0241a-ff46-4e66-b873-ae89612cd3dc)

### **After**

![image](https://github.com/user-attachments/assets/341d796f-8209-417f-991e-c994be7f9a9d)

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
